adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -646,7 +645,7 @@ trait GenomicRDD[T, U <: GenomicRDD[T, U]] extends Logging {
    * @tparam V The InFormatter to use for formatting the data being piped to the
    *   command.
    */
-  def pipe[X, Y <: GenomicRDD[X, Y], V <: InFormatter[T, U, V]](cmd: String,
+  def pipe[X, Y <: GenomicRDD[X, Y], V <: InFormatter[T, U, V]](cmd: java.util.List[String],
```

Comment:
@heuermh to your R woes, `SparkR` uses homegrown R to JVM binding code which uses Scala types. If you change this from `java.util.List[String]` to `Seq[Any]` and then cast `cmd` to `Seq[String]` (as we do with the `files` parameter), you should be good.

Reply:
Thanks! Looks like this particular comment is on the wrong line; I made the fix you recommended and tests pass for me locally.  Not sure why Jenkins is still having trouble.